src/components/blog/ImpactReportSection.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Image from "next/image";
+import annualImpactReportImg from "src/assets/annualImpactReport.png";
+import { PageSubSection } from "../common/PageSubSection";
+
+export const ImpactReportSection = () => {
+    return (
+        <PageSubSection title={"Impact Report"}>
```

Comment:
Title is supposed to be Annual Impact Report
```suggestion
        <PageSubSection title={"Annual Impact Report"}>
```